src/signals/incident/components/IncidentNavigation/index.js

```diff
@@ -16,41 +16,44 @@ const IncidentNavigation = ({ controls, meta: { wizard, submitting, handleSubmit
   const hideSubmit = controls?.hide_navigation_buttons?.meta ? controls.hide_navigation_buttons.meta.isVisible : false;
 
   return (
-    <WithWizard
-      render={({ next, previous, step }) => {
-        const currentStep = (step && step.id && step.id.split('/').pop()) || 0;
-        const wizardStep = currentStep && wizard[currentStep];
+    (wizard && (
+      <WithWizard
+        render={({ next, previous, step }) => {
+          const currentStep = (step && step.id && step.id.split('/').pop()) || 0;
+          const wizardStep = currentStep && wizard[currentStep];
 
-        return (
-          wizardStep && (
-            <div className="incident-navigation">
-              {!hideSubmit && wizardStep.previousButtonLabel ? (
-                <PreviousButton
-                  className={wizardStep.previousButtonClass}
-                  onClick={previous}
-                  data-testid="previousButton"
-                >
-                  {wizardStep.previousButtonLabel}
-                </PreviousButton>
-              ) : (
-                <span />
-              )}
+          return (
+            wizardStep && (
+              <div className="incident-navigation">
+                {!hideSubmit && wizardStep.previousButtonLabel ? (
+                  <PreviousButton
+                    className={wizardStep.previousButtonClass}
+                    onClick={previous}
+                    data-testid="previousButton"
+                  >
+                    {wizardStep.previousButtonLabel}
+                  </PreviousButton>
+                ) : (
+                  <span />
+                )}
 
-              {!hideSubmit && wizardStep.nextButtonLabel && (
-                <NextButton onClick={e => handleSubmit(e, next, wizardStep.formAction)} data-testid="nextButton">
-                  <span className="value">{wizardStep.nextButtonLabel}</span>
-                  {submitting && (
-                    <span className="working">
-                      <div className="progress-indicator progress-white"></div>
-                    </span>
-                  )}
-                </NextButton>
-              )}
-            </div>
-          )
-        );
-      }}
-    />
+                {!hideSubmit && wizardStep.nextButtonLabel && (
+                  <NextButton onClick={e => handleSubmit(e, next, wizardStep.formAction)} data-testid="nextButton">
+                    <span className="value">{wizardStep.nextButtonLabel}</span>
+                    {submitting && (
+                      <span className="working">
+                        <div className="progress-indicator progress-white"></div>
+                      </span>
+                    )}
+                  </NextButton>
+                )}
+              </div>
+            )
+          );
+        }}
+      />
+    )) ||
+    null
```

Comment:
Returning `undefined` also does the trick, right? Shouldn't be needed to explicitly return `null`

Reply:
Not really. If I do not return null, some unittests are failing. I thought a component should render something or return null, right?

`src/signals/incident/components/IncidentForm/index.test.js:68:42`
```
<IncidentForm /> › rendering › expect to render correctly

    Field(...): Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null.
```

Reply:
Ah, the illustrious incident form. Fair enough.